### PR TITLE
core: Remove cdr parser

### LINKED ISSUE #349: cdrs: Remove CDRparser.pl script
IvozProvider collects CDRs in two different ways:

**Billing related CDRs**

This stats are collected by Kamailio (KamTrunks) and are used to assign price to calls. No effort is made to correlate this calls with the user on the other leg of the call (_user/friend/retail_), the only goal is to have an stat for each call that implies cost: "company/retail X made a call to Y for Z seconds".

**'Call History' related CDRs**

CDRs above do not give any information about which user made which call and sometimes is useful to have this information (for 'Call History' and user portal). This stats are collected by both KamUsers and KamTrunks and are parsed by a periodical script call CDRparser.pl.

This issue aims to remove this script and drop ParsedCDRs table making these simplifications:

- **Only stats collected by KamUsers will be shown**: if a call is not handled by KamUsers, it won't be shown (e.g. incoming external call that is forwarded to an external destination)

- **No effort** will be made **to correlate** KamUsers leg with an eventual KamTrunks (or KamUser) leg.

- There will be two types of stats:

  - "User X dialed Y at HH:MM:SS and talked for Z seconds" stats.
  - "User X received a call from Y at HH:MM:SS and talked for Z seconds" stats. Note that "from" is the phone number shown in the phone screen when ringing (PAI/RPID/From header of initial INVITE).

- A call that has 2 (or more) legs handled by KamUsers will have 2 (or more) stats in "Call History" section.

_e.g. "1001 dials 1002" and "1002 received a call from 1001"_

- Note that call forwardings and refers won't be taken into account:

_e.g. (1) "1001 dials 1002, the call gets forwarded to 1003. 1003 transfers the call to 1004 (blindxfer)"_

3 stats: "1001 dials 1002", "1003 receives a call from 1001" and "1004 receives a call from 1001"

_e.g. (2) "1001 dials 1002, the call gets forwarded to 1003. 1003 transfers the call to 1004 (attxfer)"_

4 stats: "1001 dials 1002", "1003 receives a call from 1001", "1003 dials 1004" and "1004 receives a call from 1003"

- User portal will show calls dialed by a user and calls received by a user (the first part of the examples above).

_e.g. (1)_
1001 userportal: "1001 dials 1002" stat
1002 userportal: no stat
1003 userportal: "1003 receives a call from 1001"
1004 userportal: "1004 receives a call from 1001"

_e.g. (2)_
1001 userportal: "1001 dials 1002" stat
1002 userportal: no stat
1003 userportal: "1003 receives a call from 1001" +  "1003 dials 1004"
1004 userportal: "1004 receives a call from 1003"

**This change is not backwards compatible and implies an inherent data-loss**.

### PR DESCRIPTION
**Pull request by cruzccl**
This group of commits remove CDRparser script and all the stuff related to it:

- ParsedCDRs table.
- Sections in portals.

It rebuilds a new section called "Call history" that shows a simplified table with calls handled by KamUsers (raw data inserted by Kamailio acc module).

Needs changes related with Kamailio issues 1358 and 1378 to work.

Fixes #349

### CODE REVIEW

File: library/CoreBundle/Resources/config/lifecycle/ivoz/brand.yml [hunk 1]
```diff
@@ -31,15 +31,6 @@ services:
...
-      - { name: provider.lifecycle.brand.post_persist, priority: 20 }
-    arguments:
-      $entityPersister:
-        '@Ivoz\Core\Infrastructure\Domain\Service\DoctrineEntityPersister'
-      $usersDomainAttrRepository:
-        '@=service("doctrine.orm.entity_manager").getRepository("Ivoz\\Kam\\Domain\\Model\\UsersDomainAttr\\UsersDomainAttr")'
-
   Ivoz\Provider\Domain\Service\RoutingPattern\UpdateByBrand:
     tags:
       - { name: provider.lifecycle.brand.post_persist, priority: 30 }
```
Comment (Kaian) [inline library/CoreBundle/Resources/config/lifecycle/ivoz/brand.yml]: I would update the rest of block priorities when one is removed.

File: library/CoreBundle/Resources/config/lifecycle/ivoz/company.yml [hunk 2]
```diff
@@ -31,13 +31,6 @@ services:
...
-  Ivoz\Kam\Domain\Service\UsersDomainAttr\CreateByCompany:
-    tags:
-      - { name: provider.lifecycle.company.post_persist, priority: 20 }
-    arguments:
-      $entityPersister:
-        '@Ivoz\Core\Infrastructure\Domain\Service\DoctrineEntityPersister'
-
   Ivoz\Provider\Domain\Service\CompanyService\PropagateBrandServices:
     tags:
       - { name: provider.lifecycle.company.post_persist, priority: 30 }
```
Comment (Kaian) [inline library/CoreBundle/Resources/config/lifecycle/ivoz/company.yml]: Same here

File: kamailio/trunks/config/kamailio.cfg [hunk 3]
```diff
@@ -215,7 +215,6 @@ modparam("acc", "failed_transaction_flag", FLT_ACCFAILED)
 modparam("acc", "time_mode", 4)
 modparam("acc", "time_attr", "utctime")
 modparam("acc", "time_format", "%Y-%m-%d %H:%M:%S")
-modparam("acc", "acc_time_column", "localtime")
```
Comment (Kaian) [inline kamailio/trunks/config/kamailio.cfg]: I would fixup this into f3f3397

**Comment by Kaian** [comment]
I have added some comments. Also this PR has conflicts somehow with latest bleeding changes :-S

**Comment by Kaian** [comment]
For the packaging part in your first commit (not tagged as pkg btw) cdrparser timers are also present in these files:

- debian/ivozprovider-profile-data.install
- debian/rules

**Comment by cruzccl** [comment]
Done, thank you very much for the review :)

**Pull request merged**